orchestrator/api/api_v1/endpoints/subscriptions.py

```diff
@@ -102,14 +102,14 @@ def delete_subscription(subscription_id: UUID) -> None:
         return None
 
 
-@router.get("/parent_subscriptions/{subscription_id}", response_model=List[SubscriptionSchema])
-def parent_subscriptions(subscription_id: UUID) -> List[SubscriptionTable]:
-    return query_parent_subscriptions(subscription_id).all()
+@router.get("/in_use_by_subscriptions/{subscription_id}", response_model=List[SubscriptionSchema])
+def in_use_by_subscriptions(subscription_id: UUID) -> List[SubscriptionTable]:
+    return query_in_use_by_subscriptions(subscription_id).all()
```

Comment:
The current endpoint is `/api/subscriptions/parent_subscriptions/{subscription_id}` which was already a bit verbose as it says "subscriptions" twice. With this rename change perhaps we can shorten it to `/api/subscriptions/in_use_by/{subscription_id}`?

Reply:
And maybe we want to keep the old path for some time as well and mark it as deprecated https://fastapi.tiangolo.com/tutorial/path-operation-configuration/#deprecate-a-path-operation

This will make it easier for users to adapt